src/services/tracing.ts

```diff
@@ -238,13 +238,28 @@ export const recordExceptionInCurrentSpan = ({
   recordException(span, error, level)
 }
 
+const updateErrorForSpan = ({
+  span,
+  errorLevel,
+}: {
+  span: Span
+  errorLevel: ErrorLevel
+}): boolean => {
+  const spanErrorRank = RankedErrorLevel.indexOf(span.attributes["error.level"])
+  const errorRank = RankedErrorLevel.indexOf(errorLevel)
+
+  return errorRank >= spanErrorRank
```

Comment:
if they're equal we're just overriding another error from the same level? is that useful?

Reply:
At this point it would be down to if we favor exposing earlier or later errors in the call stack. The important thing here though would be that we're not overwriting a higher level with a lower one for our alerting.

We'd be able to get details of all the actual errors still from the `Events` section in the span, and can query with attributes like `exception.message` & `exception.type` (e.g. [here](https://ui.honeycomb.io/galoy/datasets/arvin-dev/result/tsieg7hWCGz/trace/3D9Ja83hgKA?fields%5B%5D=c_name&fields%5B%5D=c_service.name&span=954e8400b489147b) where we got one instance of `InvalidContactAlias` from an explicit `recordException` and another from the function return)